src/strtable.c

```diff
@@ -24,7 +24,7 @@
 #include <string.h>
 #include <pthread.h>
 #include "strtable.h"
-#include <deltachat/deltachat.h>
+#include <deltachat-ffi/deltachat.h>
```

Comment:
why the ffi postfix? More a rust core question and not really something about -node, but still? Also saw it on rpgp.

Reply:
Because https://github.com/deltachat/deltachat-core-rust/tree/master/deltachat-ffi

Reply:
ffi = foreign function interface afaik